manifest.yml.j2

```diff
@@ -0,0 +1,95 @@
+{%- set app_vars = {
+  'notify-api': {'NOTIFY_APP_NAME': 'api', 'disk_quota': '2G', 'sqlalchemy_pool_size': 20, 'routes': {
+      'preview': ['notify-api-preview.cloudapps.digital', 'api.notify.works'],
+      'staging': ['notify-api-staging.cloudapps.digital', 'api.staging-notify.works'],
+      'production': ['notify-api-production.cloudapps.digital', 'api.notifications.service.gov.uk'],
+    }
+  },
+  'notify-api-db-migration': {'NOTIFY_APP_NAME': 'api', 'instances': 0},
+
+  'notify-delivery-celery-beat': {'NOTIFY_APP_NAME': 'delivery-celery-beat', 'instances': 1, 'memory': '128M'},
+  'notify-delivery-worker-database': {'NOTIFY_APP_NAME': 'delivery-worker-database'},
+  'notify-delivery-worker-research': {'NOTIFY_APP_NAME': 'delivery-worker-research'},
+  'notify-delivery-worker-sender': {'NOTIFY_APP_NAME': 'delivery-worker-sender', 'instances': 1, 'disk_quota': '2G', 'memory': '3G'},
+  'notify-delivery-worker-periodic': {'NOTIFY_APP_NAME': 'delivery-worker-periodic', 'instances': 1},
+  'notify-delivery-worker-priority': {'NOTIFY_APP_NAME': 'delivery-worker-priority'},
+  'notify-delivery-worker': {'NOTIFY_APP_NAME': 'delivery-worker'},
+  'notify-delivery-worker-internal': {'NOTIFY_APP_NAME': 'delivery-worker-internal'},
+  'notify-delivery-worker-receipts': {'NOTIFY_APP_NAME': 'delivery-worker-receipts'},
+  'notify-delivery-worker-service-callbacks': {'NOTIFY_APP_NAME': 'delivery-worker-service-callbacks', 'disk_quota': '2G'},
+} -%}
+
+{%- set env_defaults = {
+  'preview': {'instances': 1},
+  'staging': {'instances': 2},
+  'production': {'instances': 2}
```

Comment:
I think this needs to be passed in as a variable set to the current instance number. Otherwise `v3-apply-manifest` will immediately scale down to the given number, which we don't want to happen in production

Reply:
decision: this can be the autoscaler's responsibility [as it is outside of deployments]. Remove instances from the manifest entirely